tests/std/tests/P0896R4_views_drop/test.cpp

```diff
@@ -161,8 +156,8 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
     }
 
     // ... with rvalue argument
-    STATIC_ASSERT(CanViewDrop<remove_reference_t<Rng>> == is_view || enable_borrowed_range<remove_cvref_t<Rng>>);
-    if constexpr (is_span<remove_cvref_t<Rng>> || is_view) {
+    STATIC_ASSERT(CanViewDrop<remove_reference_t<Rng>> == is_view || movable<remove_reference_t<Rng>>);

```

Comment:
In `P0896R4_views_all`, I saw you changed

```diff
-static_assert(CanViewAll<remove_cvref_t<Rng>> == is_view || ranges::enable_borrowed_range<remove_cvref_t<Rng>>);
+static_assert(CanViewAll<remove_reference_t<Rng>> == (is_view || movable<remove_reference_t<Rng>>) );
```

The updated lines in that file and this file look very similar, except for the parentheses on the RHS, which do affect the meaning (as equality has higher precedence than logical OR).

Is this intentional? `P0896R4_views_drop` appears to be the only file without such parentheses.